packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -1229,13 +1232,16 @@ export function TransferPanel() {
   )
 
   const transferPanelMainErrorMessage:
-    | TransferPanelMainErrorMessage
+    | string
+    | TransferPanelMainRichErrorMessage
     | undefined = useMemo(() => {
     // ETH transfers using SC wallets not enabled yet
     if (isSmartContractWallet && !selectedToken) {
```

Comment:
flagging this:

this condition has an assumption that SCW will not be on an Orbit custom fee chain
which is true now
but might change later

Reply:
Fair point. I can make this message also accept the name of the native currency.

Reply:
Fair point. I'll make this message accept the name of the asset so we can make it universal.

Reply:
Fixed in https://github.com/OffchainLabs/arbitrum-token-bridge/pull/1273/commits/68c600d87a0a5e7fcec1cd1ef745dac3313dc8b2.